src/lbaf/IO/configurationValidator.py

```diff
@@ -102,35 +146,49 @@ def validate(valid_schema: Schema, schema_to_validate: dict):
     def main(self):
         """ Main routine for the config validation. """
         # Validate skeleton
-        if self.is_valid(valid_schema=self.__skeleton, schema_to_validate=self.__config_to_validate):
+        if self.is_valid(
+            valid_schema=self.__skeleton,
+            schema_to_validate=self.__config_to_validate):
             self.__logger.info("Skeleton schema is valid")
         else:
             self.__logger.error("Skeleton schema is invalid")
-            self.validate(valid_schema=self.__skeleton, schema_to_validate=self.__config_to_validate)
+            self.validate(
+                valid_schema=self.__skeleton,
+                schema_to_validate=self.__config_to_validate)
 
         # Validate from_data/from_samplers
         if (from_data := self.__config_to_validate.get("from_data")) is not None:
             self.__logger.info("Reading from data was chosen.")
-            if self.is_valid(valid_schema=self.__from_data, schema_to_validate=from_data):
+            if self.is_valid(
+                valid_schema=self.__from_data,
```

Comment:
Does it give an error?

Reply:
Python interpreter can parse it, but this is an error: `PEP 8: E125 continuation line with same indent as next logical line`,
it looks like they are all arguments of the same function.
```Python3
            if self.is_valid(
                valid_schema=self.__from_data,
                schema_to_validate=from_data):
                self.__logger.info("from_data schema is valid")
```
We have line length up to 120 chars. It could be easily visible even with resolution 1280x1024 or 1280x720.
Should be as it was:
```Python3
            if self.is_valid(valid_schema=self.__from_data, schema_to_validate=from_data):
                self.__logger.info("from_data schema is valid")
```

Reply:
Same comment as above.

Also indent does not appear **wrong**: it does not result in a syntactic error.

Reply:
As I wrote above, Python interpreter can parse it, but it is not compliant with PEP8 and is a PEP8 violation.
`PEP 8: E125 continuation line with same indent as next logical line`